scvi/autotune/_manager.py

```diff
@@ -332,21 +332,24 @@ def _validate_reporter(
         return _reporter
 
     def _validate_resources(self, resources: dict) -> dict:
-        """Validates a resources-use specification."""
+        """Validates a resource-use specification."""
         # TODO: perform resource checking
         return resources
 
-    def _get_setup_kwargs(self, adata: AnnOrMuData) -> dict:
-        """Retrieves the kwargs used for setting up `adata` with the model class."""
+    def _get_setup_info(self, adata: AnnOrMuData) -> Tuple[str, dict]:
+        """Retrieves the method and kwargs used for setting up `adata` with the model class."""
         manager = self._model_cls._get_most_recent_anndata_manager(adata)
-        return manager._get_setup_method_args().get("setup_args", {})
+        setup_method_name = manager._registry.get("_SETUP_METHOD_NAME", "setup_anndata")
+        setup_args = manager._get_setup_method_args().get("_SETUP_ARGS_KEY", {})
```

Comment:
don't we have a constant for this and `_SETUP_METHOD_NAME` ? If so, shouldn't this just be `_SETUP_ARGS_KEY` without the quotes?

Reply:
Good catch, I forgot to take out the quotes